packages/core/src/config/babel/index.ts

```diff
@@ -60,8 +60,8 @@ const getConfig = (target: Target): TransformOptions => {
       "@babel/preset-env",
       {
         targets: targets[target],
-        useBuiltIns: "usage",
-        corejs: "3",
+        useBuiltIns: "entry",
+        corejs: "3.8",
```

Comment:
Why `3.8` instead of `3`? Just curious.

Reply:
That's what the docs say 🙂, to use the minor version instead of the major.

Reply:
Ok then 🙂